### PR TITLE
fix(scripts): fix conversion in historical price feed script

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

Decimals are off in the getMedianHistoricalPrice script.

**Summary**

Ensures all price feeds are seeded with 18 as their decimals so the `fromWei` call at the end works as expected.

**Issue(s)**

N/A

### CODE REVIEW

File: packages/core/scripts/local/getMedianHistoricalPrice.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ async function getMedianHistoricalPrice(callback) {
     let priceFeedConfig = {
       // Empirically, Cryptowatch API only returns data up to ~4 days back.
       lookback: 345600,
+      priceFeedDecimals: 18, // Ensure all prices come out as 18-decimal denominated so the fromWei conversion works at the end.
```
Comment (nicholaspai) [inline packages/core/scripts/local/getMedianHistoricalPrice.js:46]: This is neccessary because of the `fromWei` below right?

**Review by nicholaspai — APPROVED** [review]
One question then LGTM

**Review by smb2796 — APPROVED** [review]
Thanks for fixing. LGTM

File: packages/core/scripts/local/getMedianHistoricalPrice.js (hunk 1 above)
Comment (mrice32) [reply packages/core/scripts/local/getMedianHistoricalPrice.js:47]: Yep.

**Pull request merged**